Cms/ProfileBioEdit.ascx.cs

```diff
@@ -0,0 +1,2389 @@
+﻿// <copyright>
+// Copyright 2025 by Kingdom First Solutions
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Linq.Dynamic.Core;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+using NuGet;
+using Rock;
+using Rock.Attribute;
+using Rock.Data;
+using Rock.Model;
+using Rock.Web.Cache;
+using Rock.Web.UI.Controls;
+
+namespace RockWeb.Plugins.rocks_kfs.Cms
+{
+    #region Block Attributes
+
+    [DisplayName( "Edit Profile" )]
+    [Category( "KFS > CMS" )]
+    [Description( "Customized Edit Profile block to have more control over the fields and interface." )]
+
+    #endregion Block Attributes
+
+    #region Block Settings
+
+    [BooleanField(
+        "Allow Photo Editing",
+        Description = "Should you be able to edit the person photo?",
+        DefaultBooleanValue = true,
+        Key = AttributeKey.AllowPhotoEditing,
+        Order = 0 )]
+
+    [BooleanField(
+        "Display Buttons in All Panels",
+        Description = "Should the Save and Cancel button be added to each panel?",
+        DefaultBooleanValue = false,
+        Key = AttributeKey.DisplayButtonsInAllPanels,
+        Order = 0 )]
+
+    [TextField(
+        "Family Member Header",
+        Key = AttributeKey.FamilyMemberFieldsHeader,
+        Description = "The Header text for the \"Family Fields\" panel.",
+        IsRequired = false,
+        DefaultValue = "Family Members",
+        Order = 1 )]
+
+    [BooleanField(
+        "Show Family Members",
+        Description = "Should family members be shown to add/edit or not?",
+        DefaultBooleanValue = true,
+        Key = AttributeKey.ShowFamilyMembers,
+        Order = 2 )]
+
+    [TextField(
+        "Person Fields Header",
+        Key = AttributeKey.PersonFieldsHeader,
+        Description = "The Header text for the \"Person Fields\" panel.",
```

Comment:
```suggestion
        Description = "The Header text for the \"Person Members\" panel.",
```

Reply:
@guymaness2 I don't think this one needs to be updated? Unless we are changing all of them to say "Panel Header" instead. Then maybe this would be:
```suggestion
        Description = "The Header text for the \"Person\" panel.",
```
Let me know what you were thinking.